content/hi/debugging.md

```diff
@@ -0,0 +1,24 @@
+---
+title: डिबगिंग
+status: Completed
+category: संकल्पना
+---
+
+## यह क्या है
+
+डिबगिंग यह एक वांछित परिणाम प्राप्त करने के लिए कंप्यूटर प्रोग्राम, सॉफ़्टवेयर या सिस्टम से बग को खोजने और हल करने की प्रक्रिया है। 
+बग एक दोष या समस्या है जिसके कारण गलत या अप्रत्याशित परिणाम मिलते हैं।
+
+## समस्या
+
+सॉफ्टवेयर विकास एक जटिल गतिविधि है जो बग को पेश किए बिना कोड लिखना लगभग असंभव बना देती है। वे बग कोड की ओर ले जाते हैं जो निष्पादित होने पर वांछित (अपरिभाषित व्यवहार) 
```

Comment:
let's rephrase this line better: वे बग कोड की ओर ले जाते हैं जो निष्पादित होने पर वांछित (अपरिभाषित व्यवहार) के रूप में कार्य नहीं करेंगे।

Reply:
Done :)